docs/learn/learn-crowdloans.md

```diff
@@ -76,7 +77,8 @@ runtime upgrades (as determined through its own local governance).
 :::info Minimum Crowdloan Contribution
 
 The minimum balance for contributions for a crowdloan campaign is 
-currently set to {{ crowdloan_min_contribution }}. 
+currently set to 
+{{ polkadot: <RPC network="polkadot" path="consts.crowdloan.minContribution" defaultValue={50000000000} filter="humanReadable"/> :polkadot }}{{ kusama: <RPC network="kusama" path="consts.crowdloan.minContribution" defaultValue={100000000000} filter="humanReadable"/> :kusama }}. 
```

Comment:
Here is an example of what I mention in the description.  Both conditionally rendered components need to be on the same line otherwise the trailing `.` would have a space added for Polkadot only.